### PR TITLE
Update CONTRIBUTING.md for Rush workflow

### PR DESCRIPTION
**Pull request by sdalonzo**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -41,42 +42,42 @@ git clone https://github.com/priceline/design-system.git
...
+rush install
 ```
 
-Please be mindful that any deletion, or edit, of `package-lock.json`
-can cause issues in dependencies of packages within the design system.
+To update the lockfile when you change dependencies in a `package.json`:
+
+```sh
+rush update
+```
```
Comment (craigpalermo) [inline CONTRIBUTING.md:57]: My PR adds rush prepare if you want to get ahead of that one

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -85,100 +86,62 @@ We use [Plop](https://plopjs.com/) to scaffold new components rather than copyin
 The configuration allows you to create new core and package components.
 Follow the steps below to create a new component:
 
-1. Run `npm run create-component` and follow the prompts
-2. Run `npm run prepare` to tell Lerna to bootstrap your new component folder
+1. Run `rush create-component` and follow the prompts
```
Comment (craigpalermo) [inline CONTRIBUTING.md:89]: My PR will simplify this command to `rush create`, then the prompt asks "component" or "package".

**Comment by craigpalermo** [comment]
@sdalonzo lgtm overall, but we might want to wait to merge this until after #935

**Review by pcln-barry — APPROVED** [review]
Nice 👍

**Review by craigpalermo — APPROVED** [review]

**Pull request merged**